qa-covid19.planx-pla.net/manifest.json

```diff
@@ -4,25 +4,25 @@
     "That's all I have to say"
   ],
   "versions": {
-    "arborist": "quay.io/cdis/arborist:2020.03",
+    "arborist": "quay.io/cdis/arborist:2020.05",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
     "covid19-etl": "quay.io/cdis/covid19-etl:1.0.11",
     "nb-etl": "quay.io/cdis/nb-etl:1.0.11",
-    "fence": "quay.io/cdis/fence:2020.03",
-    "indexd": "quay.io/cdis/indexd:2020.03",
-    "peregrine": "quay.io/cdis/peregrine:2020.03",
-    "pidgin": "quay.io/cdis/pidgin:2020.03",
+    "fence": "quay.io/cdis/fence:2020.05",
+    "indexd": "quay.io/cdis/indexd:2020.05",
+    "peregrine": "quay.io/cdis/peregrine:2020.05",
+    "pidgin": "quay.io/cdis/pidgin:2020.05",
     "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
     "sheepdog": "quay.io/cdis/sheepdog:4.1.0",
     "portal": "quay.io/cdis/data-portal:covid19-3.1.0",
-    "tube": "quay.io/cdis/tube:0.3.28",
+    "tube": "quay.io/cdis/tube:2020.05",
```

Comment:
no, we need the latest tube

Reply:
2020.05 tag has the latest tube, 0.3.27 is included in this tag